R/bcdc-web-services.R

```diff
@@ -11,20 +11,33 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 
-#' Get data from the BC Web Feature Service
+#' Get data from the BC Web Feature Service (DEPRECATED)
 #'
-#' Pulls features off the web. The data must be available as a wms/wfs service.
+#'
+#' @inheritParams bcdc_get_data
+#' @param crs the epsg code for the coordinate reference system. Defaults to `3005`
+#'        (B.C. Albers). See https://epsgi.io.
+#'
+#' @return an `sf` object
+#'
+#' @export
+#'
+bcdc_get_geodata <- function(x = NULL, crs = 3005) {
+  message("bcdc_get_geodata is deprecated in favour of bcdc_get_data")
```

Comment:
```suggestion
  message("bcdc_get_geodata is defunct in favour of bcdc_get_data")
```

I think actually decommissioning the function means it's `defunct` instead of `deprecated`